CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -11,4 +11,158 @@ def get_ddb_connection(env:str=os.getenv('Environment', '')) -> 'botocore.client
     return ddbclient
 
 def layer_test() -> str:
-    return "hello world"
\ No newline at end of file
+    return "hello world"
+
+class Fridge:    
+    REQUIRED_FIELDS = ['display_name', 'fridge_state', 'address', 'lat', 'long']
+    TABLE_NAME = "fridge"
+    STATES ={'AK', 'AL', 'AR', 'AZ', 'CA', 'CO', 'CT', 'DC', 'DE', 'FL', 'GA',
+           'HI', 'IA', 'ID', 'IL', 'IN', 'KS', 'KY', 'LA', 'MA', 'MD', 'ME',
+           'MI', 'MN', 'MO', 'MS', 'MT', 'NC', 'ND', 'NE', 'NH', 'NJ', 'NM',
+           'NV', 'NY', 'OH', 'OK', 'OR', 'PA', 'RI', 'SC', 'SD', 'TN', 'TX',
+           'UT', 'VA', 'VT', 'WA', 'WI', 'WV', 'WY'}
+    FOOD_ACCEPTS = []#TODO: Fill this in
+    FOOD_RESTRICTIONS = []#TODO: fill this in
+    FRIDGE_TYPES = {
+        'display_name': 'S',
+        'username': 'S',
+        'fridge_state': 'S',
+        'address': 'S',
+        'instagram': 'S',
+        'info': 'S',
+        'url': 'S',
+        'neighborhood': 'S',
+        'organizer_email': 'S',
+        'tags': 'L',
+        'food_accepts': 'L',
+        'food_restrictions': 'L',
+        'lat': 'S',
+        'long': 'S',
+        'last_editted': 'N',
+        'profile_image': 'S',
+        'check_in_time': 'S',
+        'check_in_notes': 'S',
+        'check_in_status': 'S',
+        'check_in_image': 'S'
+    }
+
+    def __init__(self, db_client, fridge: dict={}):
+        self.display_name:str = fridge.get('display_name', None) #display_name must be alphanumeric (spaces are fine)
+        self.username:str = fridge.get('username', None)
+        self.fridge_state:str = fridge.get('fridge_state', None)
+        self.address:str = fridge.get('address', None)
+        self.instagram:str = fridge.get('instagram', None)
+        self.info:str = fridge.get('info', None)
+        self.url:str = fridge.get('url', None)
+        self.neighborhood:str = fridge.get('neighborhood', None)
+        self.organizer_email:str = fridge.get('organizer_email', None)
+        self.tags:list = fridge.get('tags', None)
+        self.food_accepts:list = fridge.get('food_accepts', None)
+        self.food_restrictions:list = fridge.get('food_restrictions', None)
+        self.lat:str = fridge.get('lat', None)
+        self.long:str = fridge.get('long', None)
+        self.profile_image:str = fridge.get('profile_image', None)
+        self.last_editted:int = fridge.get('last_editted', None)
+        self.check_in_time:int = fridge.get('check_in_time', None)
+        self.check_in_notes:str = fridge.get('check_in_notes', None)
+        self.check_in_status:str = fridge.get('check_in_status', None)
+        self.check_in_image:str = fridge.get('check_in_image', None)
+        self.db_client = db_client
+        if self.fridge_state is not None:
+            self.fridge_state = self.fridge_state.upper()
+
+    def has_required_fields(self) -> bool:
```

Comment:
nit: Since we're not returning any `Boolean` here, can we name this function something like `ensure_required_fields()`? Similar idealogy for the other 2 functions as well

Reply:
I am now returning booleans here, so will keep the name. But otherwise I would agree with you, thank you! :)